index.html

```diff
@@ -16,48 +18,67 @@
 
 <body>
 
- <!-- navigation code here -->
+  <!-- navigation code here -->
+  <div class="navigation">
+    <h1>SpaceWalkers</h1>
+    <nav>
+      <a href="#">Home</a>
+      <a href="#">About Us</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+      <a href="#">Sign In</a>
+    </nav>
+  </div>
 
   <div class="container home">
 
     <header class="intro">
-       <!-- image code here -->
+      <!-- image code here -->
+      <img src="img/main-header.png" alt="main header">
 
       <h2>Lift Off</h2>
-    
-      <p>As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate, that if you touched it with a finger it would fall apart. Seeing this has to change a man.</p>
+
+      <p>As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a
+        marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate,
+        that if you touched it with a finger it would fall apart. Seeing this has to change a man.</p>
     </header>
 
     <section class="space-images">
-        <img src="img/blast.png" class="blast" alt="Rocket blasting off">
-        <img src="img/sky.png" class="sky" alt="Space sky image">
-        <img src="img/launch.png" class="launch" alt="Launching rocket">
+      <img src="img/blast.png" class="blast" alt="Rocket blasting off">
```

Comment:
how of this blast img positioned on the page @mobile styling?

Reply:
I could set padding at 2% and margin: 0 auto.